pkg/stm32cmsis/Makefile

```diff
@@ -0,0 +1,35 @@
+PKG_NAME=stm32cmsis
+
+# The package URL depends on the family of the target STM32
+PKG_URL=https://github.com/STMicroelectronics/cmsis_device_$(CPU_FAM)
+
+PKG_VERSION_f0=70d2069248b18e1cce13a035abc3230abe48754f  # v2.3.4
+PKG_VERSION_f1=2948138428461c0621fd53b269862c6e6bb043ce  # v4.3.1
+PKG_VERSION_f2=2415e69642f0d0afd256ec93fa4ba1ac010b348e  # v2.2.3
+PKG_VERSION_f3=aa2333a5313f65eb1a63bd876edb455197b38f91  # v2.3.4
+PKG_VERSION_f4=4cf505380bf123941fd485b1ae01c0747dca3792  # v2.6.4
+PKG_VERSION_f7=71be130bda6963d9d8ddbb71530477eadf0be740  # v1.2.5
+PKG_VERSION_l0=78696f4f79cf51c8b32d0aa5c199a04f57ee4b27  # v1.9.0
+PKG_VERSION_l1=4a2f3cd6ea4bbc8e36daff0c7574b0fc1dfb7df7  # v2.3.0
+PKG_VERSION_l4=e442c72651e8d4757f6562acc14da949644944ce  # v1.6.1
+PKG_VERSION_wb=44490652c88a513ec9a892c328db0246cf113987  # v1.4.0
+
+PKG_VERSION=$(PKG_VERSION_$(CPU_FAM))
+PKG_LICENSE=Apache-2
+
+# Store the cloned repositories under the stm32 directory to prevent downloads
+# for each builds.
+# This is only enabled when not building on CI because to prevent issues
+# with Murdock.
+ifneq (1,$(RIOT_CI_BUILD))
+  PKG_BUILDDIR = $(RIOTBASE)/cpu/stm32/include/vendor/cmsis/$(CPU_FAM)
+endif
+
+.PHONY: all stm32cmsis
+
+include $(RIOTBASE)/pkg/pkg.mk
+
+all: stm32cmsis
+
+stm32cmsis:
+	"$(MAKE)" -C $(RIOTBASE)/pkg/$(PKG_NAME) -f $(CURDIR)/$(PKG_NAME).mk
```

Comment:
is this actually doing anything? I mean, are there sources?

Reply:
It is a header only package so indeed there's no for building an empty module: I made stm32cmsis a pseudomodule and it works the same (maybe even a little bit faster).